frontend/src/store/modules/accounts.ts

```diff
@@ -18,6 +18,7 @@ class AccountsModule extends VuexModule {
 	public activeBalance = 0
 	public chainId: string | null = null
 	public chainName: string | null = null
+	public blockExplorerUrl: string | null = null
```

Comment:
`Networks.ts` in `frontend/src/utils` has definitions of networks.
We could reuse the first item in `blockExplorerUrls`.